streamalert_cli/terraform/helpers.py

```diff
@@ -24,3 +29,33 @@ def terraform_check():
                        'your $PATH:\n'
                        '\t$ export PATH=$PATH:/usr/local/terraform/bin')
     return run_command(['terraform', 'version'], error_message=prereqs_message, quiet=True)
+
+def terraform_state_lock(action, region, table):
```

Comment:
since this function appears to be specifically related to creation and deletion of the dynamo db table, maybe pick a more appropriate/descriptive function name?

Reply:
I think I'm just going to break it out into two functions. It will make the logic cleaner and easier to name the function

Reply:
sgtm